api_tests/preprints/views/test_preprint_citations.py

```diff
@@ -128,15 +128,36 @@ def setUp(self):
         self.third_contrib.family_name = 'Schematics'
         self.third_contrib.save()
 
+        self.MLA_DATE_FORMAT = '%-d {month} %Y'
+
+        # MLA month abreviations here
+        #  http://www.pomfret.ctschool.net/computer_classes/documents/mla-abbreviationsofmonths.pdf
+        self.MLA_MONTH_MAP = {
```

Comment:
Woops, my comment got lost. Can you just do the same formatting that the citeproc code is doing instead of reimplementing it here?

Reply:
Citeproc pulls this info from the MLA's csl file, [here](https://github.com/citation-style-language/styles/blob/master/modern-language-association.csl#L24-L38). I don't really think that's practical here.

Reply:
To clarify, I meant use whatever routine we're using to render the citation, then assert that the api is returning that.

```python
citation = res.json['data']['attributes']['citation']
assert citation == render_citation(preprint, ...)
```